src/common/device_helpers.cuh

```diff
@@ -268,7 +269,7 @@ public:
     LOG(CONSOLE) << "======== Device " << current_device << " Memory Allocations: "
       << " ========";
     LOG(CONSOLE) << "Peak memory usage: "
-      << stats_.peak_allocated_bytes / 1000000 << "mb";
+      << stats_.peak_allocated_bytes / 1048576 << "mb";
```

Comment:
Probably should change the `mb` to `MiB`.

Reply:
done